roles/foreman_client_repositories/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: 'Foreman Client Staging Repository'
+  yum_repository:
+    name: foreman-client-koji
+    description: "Foreman {{ foreman_client_repositories_version }} Client Koji Repository"
+    baseurl: "http://koji.katello.org/releases/yum/foreman-client-{{ foreman_client_repositories_version }}/el{{ ansible_distribution_major_version }}/x86_64/"
+    priority: 1
```

Comment:
For my own knowledge what does "Enforce ordered protection of repositories" mean? Is this the order in which yum would go through repositories to find a particular package?

Reply:
Yep